botany/src/main/java/binnie/botany/modules/ModuleFlowers.java

```diff
@@ -112,7 +113,7 @@ public void registerItemsAndBlocks() {
 		Botany.proxy.registerItem(pollen);
 		Botany.proxy.registerItem(seed);
 
-		BinnieCore.getBinnieProxy().registerTileEntity(TileEntityFlower.class, "botany.tile.flower");
+		BinnieCore.getBinnieProxy().registerTileEntity(TileEntityFlower.class, new ResourceLocation("botany:tile.flower"));
```

Comment:
Is this backwards compatible? Will TEs disappear from worlds if they are already placed due to the name change?

Reply:
Looks like it's not backwards compatible, unfortunately

Reply:
Forestry had the same issue a while ago and we solved it with a datafixer: https://github.com/ForestryMC/ForestryMC/commit/9b4a4c2b1d0b495f180a4dfea2bcfe492546b4d0. Just have to be careful to test all the TEs are preserved

Reply:
Great suggestion! Thanks for pointing that out to me.

Reply:
The entities are backwards compatible now.